package de.aittr.g_37_jp_shop.exception_handling.exceptions;

public class FourthTestException extends RuntimeException{

    public FourthTestException(String message) {
        super(message);
    }

    public FourthTestException(String message, Throwable cause) {
        super(message, cause);
    }
}
